meetings/2025-02-05.md

```diff
@@ -28,18 +35,89 @@
+
+https://github.com/solid/specification/issues/708
+
+* eP: I will have a call with Sebastien right after this call. We could use few minutes in that call to coordinate next steps.
+* eP: I would like to decide if we take it as a group item. If we create a repo on Friday social web CG will have a meeting. As PAC recommended last week, we should coordinate with the Social Web group.
+* SR: nothing to add, it would be great if a repository could be created.
+* HZ: we can make the decision now. Any objections? (None)
+* Rahul: It's a good idea to formalize that with a repo.
+
+ACTION: elf Pavlik and Hadrian to work with W3C and Jesse to get the solid/activitypub-interop repository created.
+
 ### Wallet API
 * (continuation if MdJ is present; MdJ mentioned work with Dutch universities)
+* MdJ not present. Will delay until next week.
+
+### ShapeRepo
+
+* SR: On Jesse's suggestion, I'd like to propose a Special Topic Meeting about a shared repository for shapes and shapetrees.
+* HZ: It's part of Solid mission to have such repositories.
+* SR: First to have a discussion and see where to go from there. I'll write a post on the Solid forum with requirements that I see.
+  I think we need to have at least Jesse and Jackson Morgan.
+* Rahul: Michiel de Jong has collated work on this, you might want to include it as well. https://pdsinterop.org/conventions/
+* Jesse: You should look at the [solid/roadmap repo](https://github.com/solid/roadmap)
+
+ACTION: SR to prepare a short document with requirements and organize the STM meeting, with a poll to ensure people can be present.
+
+* RG: Are you still going forward with that grant proposal?
+* Jesse: There was a bounty proposal system, reviewers didn't understand how shapes work in Solid. I spoke with NLnet at FOSDEM, they are keen to support Solid and we could rework the grant and get it fast forwarded. We can discuss it as part of the STM.
+* eP: Will have a call with NLnet soon about a grant I asked before Oct 1st and it moved to phase 2, I will mention this to them.
+
+### Insertion with Blank node with N3 Patch
+
+https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1968
+
+https://github.com/solid/specification/issues/704
+
+* RG: Context: Migrating from NSS to CSS. NSS supports SPARQL update and N3 Patch, but CSS only supports N3 Patch. I don't know the context of the discussion, but want to know what I can do to make SolidOS work as it may be broken with only N3 Patch
+* Jesse: This seems like a discussion for the N3 CG, not the Solid CG.
+* eP: N3 is not so mature, can we really use it as a dependency?
+* Jesse: N3 is thinking about becoming a WG.
+* eP: N3-patch only exist within the Solid spec, so what will happen with the new WG ?
+* RG: We are not telling the WG to include this, but we are making them aware that there are some problems (?)
+* Jesse: If you have a clear solution in mind, no one is going to object that, and the WG will take this as a valuable input.
+* ...
```

Comment:
```suggestion
* RG: Context: Migrating from NSS to CSS. NSS supports SPARQL Update and N3 Patch, but CSS only supports N3 Patch. I don't know the context of the discussion, but want to know what I can do to make SolidOS work, as it may be broken with only N3 Patch.
* Jesse: This seems like a discussion for the N3 CG, not the Solid CG.
* eP: N3 is not so mature. Can we really use it as a dependency?
* Jesse: N3 is thinking about becoming a WG.
* eP: N3-patch only exists within the Solid spec, so what will happen with the new N3 WG?
* RG: We are not telling the WG to include this, but we are making them aware that there are some problems. (?)
* Jesse: If you have a clear solution in mind, no one is going to object to that, and the N3 WG will take this as a valuable input.
* ...
```